src/Parallel/Tags/ExitCode.hpp

```diff
@@ -0,0 +1,29 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+
+namespace Parallel::Tags {
+
+/*!
+ * \brief Exit code of the executable
+ *
+ * The following exit codes are defined:
+ *
+ * - 0: Program is complete.
+ * - 1: Program aborted because of an error.
+ * - 2: Program is incomplete and should be continued from the last checkpoint.
+ *
+ * \par Design decisions
+ * - We implement the exit code as an `int` instead of an enum because its
```

Comment:
Why not have the values be hard-coded in an `enum` and then convert them to an `int` where we need them to be an `int`? I worry that the current approach requires us to match something in this documentation block. E.g.
```cpp
enum class ExitCode : int {
  Complete = 0,
  Abort = 1,
  ContinueFromCheckpoint = 2
};
```
and then
```cpp
static_cast<int>(ExitCode::ContinueFromCheckpoint)
```
If you choose to make the `enum` not an `enum class` you can just do an implicit cast.

Reply:
I would also prefer using enums in this manner...

Reply:
ok done!